### PR TITLE
Add initial readme instructions

### LINKED ISSUE #3: Add initial README instructions
Add initial README instructions for contribution and workflow.

A few basic commands is ok, for now. 

We can have newcomers work on it, later, to help get them involved right away.

### PR DESCRIPTION
**Pull request by Felistas**
Fixes #3 

Screenshot:
Contribution.md Preview

screen shot 2019-01-10 at 13 09 33

### CODE REVIEW

**Comment by joaopapereira** [comment]
When you say WIP do you intend to keep on working?
Do you want early feedback?

**Comment by FedericoEsparza** [comment]
@Felistas Thank you for jumping in and working on this ticket.
 
What I would like to see, as part of the workflow instructions, is a variant--if not exactly--of the following:

  * Once you decide on an issue to work on (look for the *Help Wanted* or *Good First Issue* tags), create a branch from the terminal with the issue number and a short description, for example:

    ```js
    //Be sure you create your branch from an up-to-date develop branch
    //Don't modify develop. Keep it clean & in sync with this Github repo's develop branch
    git checkout develop

    //Update develop branch
    git pull

    //Create feature branch
    git checkout -b 17-add-logo
    ```

* When you're ready to submit your changes in a pull-request, first update your develop & your feature branch:
    ```js
    git checkout develop

    git pull

    //Switch back to your feature branch
    git checkout 17-add-logo

    //Update your feature branch by merging develop
    git merge develop
    ```
Keep in mind that if you write instructions for forking the repo, they would have to run `git pull upstream develop`, instead of just `git pull`, so you would also want to add instructions on how to add the base Agileventures/Websiteone-FE repo as a remote.
 
In other words, tell them to run something like this command:
```sh
    git remote add upstream https://github.com/AgileVentures/WebsiteOne-FE
```

**Comment by Felistas** [comment]
Yes @joaopapereira  there are some few things I want to add. I wouldn't mind feedback too. @FedericoEsparza I will definitely implement that.

File: CONTRIBUTION_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+### Choosing Stories/tickets.
+
+When deciding on an issue to work on, look for the  `Help Wanted` or `Good First Issue` tags.
+
+### Branch Naming Conventions
+
+Kindly follow these conventions when creating branches and pull requests.
+```
```
Comment (joaopapereira) [inline CONTRIBUTION_GUIDE.md:8]: I would move this to a separated topic, and move the actual convention under this.
Maybe this part could be under the title 
How to create a feature branch

**Review by joaopapereira — CHANGES_REQUESTED** [review]
The rest looks pretty good

File: CONTRIBUTION_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+git pull upstream develop
+``` 
+depending on the name of your origin.
+
+You will now have the latest copy of develop in your local.
+
+After that run 
+```
+git merge upstream develop
```
Comment (FedericoEsparza) [inline CONTRIBUTION_GUIDE.md]: @Felistas Would you mind explaining the purpose of this command `git merge upstream develop`? I think it might end up confusing newcomers, as it has confused me. Might it not be necessary, since we already ran `git pull upstream develop`?

**Comment by FedericoEsparza** [comment]
@Felistas I agree with @joaopapereira . The rest is looking great. I like the initiative to create the `CONTRIBUTION_GUIDE.md`
 
Please see my additional comment on that file.

File: CONTRIBUTION_GUIDE.md (hunk 2 above)
Comment (Felistas) [reply CONTRIBUTION_GUIDE.md]: @FedericoEsparza I don't think its necessary.You are right.

File: CONTRIBUTION_GUIDE.md (hunk 1 above)
Comment (Felistas) [reply CONTRIBUTION_GUIDE.md:8]: @joaopapereira Fixed.

**Review by joaopapereira — APPROVED** [review]
Looks good going to merge
